tasks.test.js

```diff
@@ -33,4 +38,40 @@ describe('Add and remove Tasks', () => {
       ]),
     ).toStrictEqual([]);
   });
+
+  it('Update the task', () => {
+    expect(
+      updateTask('Pay the bills', 0, [
+        {
+          index: 0,
+          description: 'Paying water bills',
+          completed: false,
+        },
+      ]),
+    ).toStrictEqual([
+      {
+        index: 0,
+        description: 'Pay the bills',
+        completed: false,
+      },
+    ]);
+  });
+
+  it('Update complete status', () => {
+    expect(
+      updateStatus(false, 0, [
+        {
+          index: 0,
+          description: 'Paying water bills',
+          completed: false,
+        },
+      ]),
+    ).toStrictEqual([
+      {
+        index: 0,
+        description: 'Paying water bills',
+        completed: true,
+      },
+    ]);
+  });
```

Comment:
You did an awesome job with your tests! Kindly make sure to apply all the required tests. As requested in the project requirements: 
![image](https://user-images.githubusercontent.com/42799579/207926403-ba32a1b2-336c-4ede-bc40-c9970628544c.png) 

Currently, your project doesn't contain any mock for the localStorage functions. Please make sure to create a mock and test them 😉. This will save you time when updating your project functionality and make the application easier to scale 😄.